controllers/events.js

```diff
@@ -17,12 +18,40 @@ router.get('/event', function(req, res){
 
         // 'All' returns the entire events list
         if(req.query.type === "all") {
-            response = festivalData;
-            response.ok = true;
+
+            //Get all events from Skiddle API
+            var options = { method: 'GET',
+              url: api_keys.skiddle.url + 'events/',
+              qs: 
+               { eventcode: 'FEST',
+                 order: '4',
+                 api_key: api_keys.skiddle.key } };
+
+            console.log("GET " + options.url);
+
+            //Send the request
+            request(options, function (error, reqResponse, body) {
+              if (error) throw new Error(error);
+              var contents = JSON.parse(body);
+              if (contents.error !== 0) {
+                console.log('ERROR ' + contents.errorcode + ': ' + contents.errormessage);
+              } else {
+                  response = contents.results;
```

Comment:
can we cache the results, so when we refresh the page we don't have to hit skiddle everytime? only get the data the first time the /events endpoint is hit?